Languages/IronPython/Tests/interop/net/derivation/test_property_override.py

```diff
@@ -78,7 +77,10 @@ def get_StrProperty(self):
     x = C()
     p = IProperty11.StrProperty
     p.__set__(x, 'abc')     # no-op equivalent?
-    AssertError(SystemError, lambda: p.SetValue(x, 'def'))  # ?
+    if is_netstandard: # no System.SystemException in netstandard1.6
```

Comment:
There are a number of workarounds to get things running that we can rip out once we move to netstandard2.0 (or netstandard1.7 I'm not exactly sure about what's coming up). I will have to go through all my changes and make a list. I'll file an issue relating to `SystemError`.